source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -848,9 +891,14 @@ FilterTrailersStatus Filter::onTrailers(ProcessorState& state, Http::HeaderMap&
   state.setTrailers(&trailers);
 
   if (state.callbackState() != ProcessorState::CallbackState::Idle) {
-    ENVOY_LOG(trace, "Previous callback still executing -- holding header iteration");
-    state.setPaused(true);
-    return FilterTrailersStatus::StopIteration;
+    if (state.bodyMode() == ProcessingMode::FULL_DUPLEX_STREAMED) {
+      ENVOY_LOG(trace, "Body mode is FULL_DUPLEX_STREAMED, sending trailers even though Envoy is "
```

Comment:
I feel like this p;attern of conditionally issuing `trace` messages could be factored out somehow. WDYT?

Reply:
done